tests/integration/__init__.py

```diff
@@ -372,7 +372,7 @@ def _id_and_mark(f):
 # 1. unittest doesn't skip setUpClass when used on class and we need it sometimes
 # 2. unittest doesn't have conditional xfail, and I prefer to use pytest than custom decorator
 # 3. unittest doesn't have a reason argument, so you don't see the reason in pytest report
-requires_collection_indexes = pytest.mark.skipif(SCYLLA_VERSION is not None and Version(SCYLLA_VERSION.split(':')[1]) < Version('5.2'),
+requires_collection_indexes = pytest.mark.skipif(SCYLLA_VERSION is not None and (len(SCYLLA_VERSION.split('/')) != 0 or Version(SCYLLA_VERSION.split(':')[1]) < Version('5.2')),
```

Comment:
I understand that we have to use unstable Scylla in order to tests tablets, so this code needs to deal with unstable version, but
- It should have a TODO to remove it when we stop using unstable version in CI.
- Ideally, it could check if this unstable version is new enough to have collection indexes, but that's and overkill here probably. It would be necessary if we were to continously support unstable versions here.

Reply:
TODO added